WebRoot/js/zimbraMail/mail/view/ZmMailMsgView.js

```diff
@@ -164,7 +164,9 @@ function() {
 	if (this._objectManager && this._objectManager.reset) {
 		this._objectManager.reset();
 	}
-	this.setScrollWithIframe(this._scrollWithIframe);
+	//Commenting this line for ZCS-8165 as need to set iframe scroll auto for all messages 
+	//this.setScrollWithIframe(this._scrollWithIframe);
+	this.setScrollWithIframe("auto");
```

Comment:
I think this should fix the issue in more framework compliant way
`this._scrollWithIframe ? DwtControl.CLIP : DwtControl.SCROLL`

Reply:
I was thinking same but unfortunately its not working.